threetenabp/src/main/java/com/jakewharton/threetenabp/format/AndroidDateTimeFormatter.java

```diff
@@ -0,0 +1,62 @@
+package com.jakewharton.threetenabp.format;
+
+import android.content.Context;
+import android.content.res.Configuration;
+import android.os.Build;
+import android.os.LocaleList;
+
+import org.threeten.bp.chrono.IsoChronology;
+import org.threeten.bp.format.DateTimeFormatter;
+import org.threeten.bp.format.DateTimeFormatterBuilder;
+import org.threeten.bp.format.FormatStyle;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+
+/**
+ * Provides Android-specific {@link DateTimeFormatter}s, such as a localized time formatter that
+ * respects the user's 12-/24-hour clock preference.
+ */
+public class AndroidDateTimeFormatter {
```

Comment:
final

Reply:
Happy to do this, but I noticed that `android.text.format.DateFormat` is not final. I suppose correctness is more important than consistency here.